xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-api/src/main/java/org/xwiki/ratings/internal/averagerating/XObjectAverageRatingManager.java

```diff
@@ -32,17 +35,23 @@
 import org.xwiki.component.annotation.Component;
 import org.xwiki.component.annotation.InstantiationStrategy;
 import org.xwiki.component.descriptor.ComponentInstantiationStrategy;
-import org.xwiki.model.EntityType;
 import org.xwiki.model.reference.EntityReference;
 import org.xwiki.model.reference.EntityReferenceSerializer;
 import org.xwiki.properties.converter.Converter;
 import org.xwiki.ratings.AverageRating;
 import org.xwiki.ratings.RatingsException;
+import org.xwiki.ratings.events.UpdateAverageRatingFailedEvent;
+import org.xwiki.ratings.events.UpdatedAverageRatingEvent;
+import org.xwiki.ratings.events.UpdatingAverageRatingEvent;
 
 import com.xpn.xwiki.XWikiContext;
+import com.xpn.xwiki.XWikiException;
 import com.xpn.xwiki.doc.XWikiDocument;
 import com.xpn.xwiki.objects.BaseObject;
 
+import static org.xwiki.model.EntityType.DOCUMENT;
+import static org.xwiki.model.EntityType.WIKI;
```

Comment:
Here it's really a matter of style, but personally I prefer to import the enum type and not the enum values. The reason is that `WIKI` in the code can be anything, while `EntityType.WIKI` is more explicit IMO.